nengo_loihi/loihi_api.py

```diff
@@ -313,7 +313,7 @@ def add_synapses(self, synapses):
         a0 = 0
         if len(self.synapse_axons) > 0:
             last = next(reversed(self.synapse_axons))
-            a0 = self.synapse_axons[last][-1]
+            a0 = self.synapse_axons[last][-1] + 1
```

Comment:
Two ways: if `self.synapse_axons[last][-1]` is the last index we used, then we need to add 1 to get the next index. I came across it because I was looking at the compartments actually created and their indexes, and noticing that for groups with multiple synapses objects things weren't lining up.

We could write a hyper-specific test that looks at this. However, I consider this as one of a family of bugs where either: we overwrite things on the N2Board that we've already configured, or we miss configuring objects that we expect to have configured. Unfortunately, NxSDK does not make it easy to detect either of these things. I think the biggest start would be adding some "static" (not running anything on the chip) tests for `loihi_api` and `allocator`, which we've talked about doing, but has never been done.